docker/driverloader/Dockerfile

```diff
@@ -0,0 +1,14 @@
+ARG FALCO_VERSION=latest
+FROM falcosecurity/falco:${FALCO_VERSION}
```

Comment:
Can we create a two step image here so to have the `falcosecurity/driverloader` slimmer?

Reply:
Another approach could be to start from scratch and extract the `falco-driver-loader` script from the `bin` (or `bin-dev`) release...

Reply:
WDYT

Reply:
Actually, I have used `FROM falcosecurity/falco:${FALCO_VERSION}` in order to include the toolchain, not just `falco-driver-loader`. 

Alternatively, to slim down the image, we could start from scratch, include `falco-driver-loader` - as you proposed - and add just the dependencies required by `falco-driver-loader`, but it would mean maintaining a separate dependencies list.